receiver/sqlserverreceiver/README.md

```diff
@@ -14,29 +14,45 @@
 [sumo]: https://github.com/SumoLogic/sumologic-otel-collector
 <!-- end autogenerated section -->
 
-The `sqlserver` receiver grabs metrics about a Microsoft SQL Server instance using the Windows Performance Counters.
-Because of this, it is a Windows only receiver.
+The `sqlserver` receiver grabs metrics about a Microsoft SQL Server instance. The receiver works by either using the
+Windows Performance Counters, or by directly connecting to the instance and querying it. Windows Performance Counters
+are only available when running on Windows.
 
 Make sure to run the collector as administrator in order to collect all performance counters for metrics. 
 
 ## Configuration
 
 The following settings are optional:
 - `collection_interval` (default = `10s`): The internal at which metrics should be emitted by this receiver.
+- `instance_name` (optional): The instance name identifies the specific SQL Server instance being monitored.
+  If unspecified, metrics will be scraped from all instances. If configured, the `computer_name` must also be set
+  when running on Windows.
+
+Direction connection options (optional, but all must be specified to enable):
+- `username`: The username used to connect to the SQL Server instance.
+- `password`: The password used to connect to the SQL Server instance.
+- `server`: IP Address or hostname of SQL Server instance to connect to.
+- `port`: Port of the SQL Server instance to connect to.
```

Comment:
`server` and `port` here are separate options for the sake of the SQL db connection string, which requires them to be passed in separately. I thought this makes more sense than the alternative of a single option, and then parsing internally. Happy to discuss if anyone prefers the alternative.